cfgov/v1/atomic_elements/organisms.py

```diff
@@ -962,7 +962,7 @@ class Meta:
 
     class Media:
         js = ["mortgage-performance-trends.js"]
-        css = ["mortgage-performance-trends.css"]
+        css = ["mortgage-performance-trends.css", "chart.css"]
```

Comment:
Actually, would it make more sense to import `chart.less` inside `mortage-performance-trends.less`?

Reply:
I think it would be preferable to keep them separate, so they can be cached separately and change on different cadences without breaking the cache of the other.